### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by twerthi**
# About

This is a post about demonstrating how to deploy to Azure Container Apps using Octopus Deploy.

# Release

Is this post for a specific release? If so which one?

# Blog image idea

I'd imagine something containing the Azure Container Apps logo, artists choice :)

# Prerequisites

- [X] The draft is complete and this post is ready to be reviewed.
- [X] I'm familiar with the [guide for blog content basics](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content) and [writing educational content](https://www.octopus.design/932c0f1a9/p/137b06-educational-content---how-we-write-about-what-we-do).
- [X] I've checked the content using Grammarly (Octopus has a corporate license) and the [Hemingway app](https://hemingwayapp.com/) to remove typos, grammatical errors, and passive voice. 
- [X] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://www.octopus.design/932c0f1a9/p/901d2a-blog-content/t/817249) and none of those apply.
- [X] The images I've included follow the [rules for working with images](https://www.octopus.design/932c0f1a9/p/5061d7-working-with-images).
- [X] I've added alt text for all the images I've included (alt text describes an image to people unable to see it, 125 characters max)
- [X] I've drafted social media posts for Twitter and LinkedIn and will post them for review in #topic-social-media.
- [X] I've dropped a note in Slack in #topic-content-marketing to let Tegan Ali know my post is ready for review. (No need to fill out a content brief.)

## How to review this PR

If there's anything you'd like reviewers to focus on, mention it here.

## Publication date

If there are considerations for when to publish this post, mention them here. For example, this post is supporting material for a webinar I'll be conducting on date, or this post should not published until after a specific release on date.

### CODE REVIEW

File: blog/2023-07/azure-container-apps/index.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+# Import modules
+Import-Module Az.App
+
+# Get reference to the container
+$apiContainerApp = Get-AzContainerApp -Name "$($OctopusParameters["Project.Azure.ContainerApp.Api.Name"])" -ResourceGroupName $OctopusParameters["Project.Azure.ResourceGroup.Name"]
+
+Set-OctopusVariable -name "DNS" -value $apiContainerApp.IngressFQDN
+```
+
+#### Azure - Deploy web container app
```
Comment (TeganAli) [inline blog/2023-07/azure-container-apps/index.md:241]: @twerthi should this heading be "Azure - Deploying the web container app"?

Comment (twerthi) [reply blog/2023-07/azure-container-apps/index.md:241]: Those were the step names, but if it doesn't make sense, we can change them.

Comment (TeganAli) [reply blog/2023-07/azure-container-apps/index.md:241]: Oh my apologies. I see now. I think if I add "step" on the end that will make it clearer. I'll edit them now but feel free to let me know if you don't think that works. Thanks.

File: blog/2023-07/azure-container-apps/index.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+- Azure Account Tenant Id: `#{Project.Azure.Account.TenantId}`
+- Azure Account Password: `#{Project.Azure.Account.Password}`
+- Container App Environment Name:  `#{Project.Azure.ContainerApp.Environment.Name}`
+- Azure Location: `#{Project.Azure.Region.Code}`
+
+![Azure - Create Container App Environment](octopus-azure-create-environment.png)
+
+The template sets an output variable called `ManagedEnvironmentId` which is the ID value of the environment it created or found if it already exists.
+
+#### Azure - Deploy api Container App step
```
Comment (TeganAli) [inline blog/2023-07/azure-container-apps/index.md:169]: @twerthi is this the correct name of the step? Or should we delete api? The link below this heading goes to a step called "Azure - Deploy Container App" not "Azure - Deploy api Container App step". (Although the screenshot of shows the step with the name "Azure - Deploy api Container App".)

Comment (twerthi) [reply blog/2023-07/azure-container-apps/index.md:169]: Configuring the deployment process lists the steps in the deployment process by what I named them, so I was trying to follow that.  The link goes to a newly added Community Step Template, I was attempting to highlight its existence.  Do you feel it's confusing to the reader?  I'm happy to adjust, but wanted to explain what it was I am doing.

**Pull request merged**